db/migrate/20170209203904_orchestration_stack_enhancements_for_ansible_service_provision.rb

```diff
@@ -0,0 +1,14 @@
+class OrchestrationStackEnhancementsForAnsibleServiceProvision < ActiveRecord::Migration[5.0]
+  def change
+    add_column :orchestration_stacks, :playbook_name, :string
```

Comment:
Most of these fields will contain data that isn't accessible from the database at the moment.  I added "playbook_name" and "project_name" (currently available through relations) to cache that data on the Job, saving 3 DB queries per reload.  I can see a user refreshing the screen many times during a provision and the binary blob lookup will be bad enough.